### PR TITLE
Add traits button to attributes display

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- Add traits button to unit attributes display - medic, engineer, eod
- Compatible with ACE medical, repair, and  explosives

### CODE REVIEW

File: addons/attributes/stringtable.xml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project name="ZEN">
     <Package name="Attributes">
+        <Key ID="STR_ZEN_Attributes_Medic">
+            <English>Medic</English>
+        </Key>
```
Comment (dedmen) [inline addons/attributes/stringtable.xml]: From languagecore_f

```
<Key ID="str_support_medic">
  <Original>Medic</Original>
  <French>Infirmier</French>
  <Spanish>Médico</Spanish>
  <Italian>Medico</Italian>
  <Polish>Medyk</Polish>
  <Russian>Санинструктор</Russian>
  <German>Sanitäter</German>
  <Czech>Zdravotník</Czech>
  <English>Medic</English>
  <Chinese>醫護兵</Chinese>
  <Portuguese>Médico</Portuguese>
  <Korean>의무병</Korean>
  <Chinesesimp>军医</Chinesesimp>
  <Turkish>Sıhhiye</Turkish>
  <Japanese>医者</Japanese>
</Key>
```

**Comment by CreepPork** [comment]
I'm starting to think, you have a separate entry for the medic attribute. But what then happens if you want to add an engineer slot and EOD (from ACE)? Are you going to create 2 more rows? I think that will take up too much space and make it cluttered.

**Comment by mharis001** [comment]
I don't think there is a way to avoid 3 rows of attributes for: medic, engineer, and EOD. Maybe checkboxes instead of toolboxes would look better.

**Comment by CreepPork** [comment]
What about a button like `Inventory` for the skills?

**Comment by jonpas** [comment]
Mind, not all are Yes/No, but have 3 or more step setting.

**Comment by mharis001** [comment]
Yeah, will need to add special handling for ACE. I'm not sure on putting this under a "buttons display". It would make the main attributes display cleaner but add an extra step in accessing this attribute.

**Comment by CreepPork** [comment]
It's not like you are going to be selecting an entire squad of units to set as medics. It doesn't have to get accessed very often. You could append this to #42.

**Pull request merged**